content/chapter-03/SystemMetricsLab.md

```diff
@@ -1 +1,40 @@
-# Lab: common tools for system metrics
\ No newline at end of file
+# Lab: common tools for system metrics
+
+* In this lab, two Virtual Machine instances will be created: One will run a stress test, and the other will run the application to be tested.
```

Comment:
In this lab, we will use two Virtual Machines: one to run a stress test from, and the second instance to run the application being tested on.

Reply:
Done